### ISSUE TITLE
Modding Harvesters?

### ISSUE THREAD

**Issue by JimmyBeam1990** [yumi-modding/FS22_CombineXP#29]
I noticed that modded combines are already present in combineXP.xml, just models from the modhub, but how do I add new models? simply via the path data/....... or with the tag $moddir$, which variant is the right one?

**Comment by yumi-modding** [comment]
Hello @JimmyBeam1990 
For mods, you need to use the "$moddir$" syntax like for the $moddir$FS22_newHollandTX32/tx32/tx32.xml
To give you more information, the need of adding mods in this file is only to be able to override the default basePerf computation that is base on combine horse power. The tx32 in the file is just as example since setting basePerf to 0 leads to using the default computed value.

**Comment by JimmyBeam1990** [comment]
all right, thank you for the answer

But I noticed that the New Holland https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=226342&title=fs2022 is available in the script and works, but it is a mod and not available in the original game? 

<vehicle xmlPath="data/vehicles/newHolland/cr1090/cr1090.xml" basePerf="963"/>

**Comment by yumi-modding** [comment]
Yes every vehicle with the combine specialization will work without setting anything in this file. The file enable players to set different performance if they want. So all base game vehicles + some giants mods are listed so it's easier to edit them, but any mod vehicle can be added also following the same model.
Also i think some vehicles listed at the end might be with file path from fs19 and might be useless.
